server/src/main/java/com/projects/pms/models/Deductions.java

```diff
@@ -0,0 +1,34 @@
+package com.projects.pms.models;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Deductions")
+@Data
+public class Deductions {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    @Column(name = "employeeid")
+    private Integer employeeId;
+    @Column
+    private double pf;
```

Comment:
@SunilSunny698 Have an proper and understandable column names

Reply:
Done....